package/Dockerfile

```diff
@@ -36,7 +38,7 @@ RUN export GRPC_HEALTH_PROBE_DOWNLOAD_URL=$(wget -qO- https://api.github.com/rep
     chmod +x /usr/local/bin/grpc_health_probe
 
 # Stage 2: build binary from c source code
-FROM registry.suse.com/bci/bci-base:15.7 AS cbuilder
+FROM registry.suse.com/bci/bci-base:15.6 AS cbuilder
```

Comment:
How does reverting `bci-base` to `15.6` help resolving the instability issue?

Reply:
Reverting bci-base to 15.6 is due to it is a Tech Preview version.
For the stability, it is addressed in https://github.com/longhorn/longhorn-instance-manager/pull/910/commits/f266aee1fcb5194f9703de08ef6f46af8cbb30bc.